src/Kestrel.Core/Internal/Http2/Http2Frame.cs

```diff
@@ -66,5 +67,21 @@ public int StreamId
         }
 
         public Span<byte> Payload => new Span<byte>(_data, PayloadOffset, Length);
+
+        public void TraceFrame(ILogger logger, string connectionId)
```

Comment:
Where is this called?

Reply:
From a different change that I didn't clean up. Will remove.